go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -326,18 +330,26 @@ func processActiveScenarioUpdate() {
 						throughputUL = ue.NetChar.ThroughputUl
 					}
 
+					//update measurements, don't wait for ticker
+					ueMeasMap, _ := sbi.gisCache.GetAllMeasurements()
```

Comment:
We should call this outside the UE iteration loop if there is at least 1 UE.

Reply:
good catch, fixed